### DISCUSSION TITLE
查询延迟很高，所有节点资源使用的并不多

### DISCUSSION THREAD

**Discussion by qiqiqi998** [milvus-io/milvus#38668] [Q&A and General discussion] [answered]
milvus的查询延迟很高，但是我并没有很高的并发。
![图片](https://github.com/user-attachments/assets/6bb34eeb-18fd-49d5-add4-be290d563ab2)
![图片](https://github.com/user-attachments/assets/e1cbc8bb-f7c5-4960-8d6f-445e502272d7)
![图片](https://github.com/user-attachments/assets/a4dfc068-c4b7-4d4c-922d-a7d89ad07e04)
![图片](https://github.com/user-attachments/assets/ed657b1e-6037-4441-bb8a-744ec6e2c0a1)
![图片](https://github.com/user-attachments/assets/5cdceadd-6305-4bb1-b95b-e111ad4c685e)
![图片](https://github.com/user-attachments/assets/be6218bd-ca71-442e-a7b4-852051d95f3a)

**Comment by qiqiqi998** [comment]
节点时间不一致导致
![图片](https://github.com/user-attachments/assets/c8f4754a-0b75-45af-9b26-d724e48160f4)

**Reply by yhmo** [reply, marked as answer]
有一台proxy的时间跟mixcoord的时间相差14秒，这个proxy如果收到查询请求，该请求的时间就会领先于mixcoord所决定的一致性时间
search默认的consistency level是Bounded，Bounded默认容忍5秒的不一致，所以该请求就会等待大约14-5=9秒
你把蓝线里的这个proxy的时间调整为跟mixcoord一致，这样就可以避免这个问题：
![Screenshot from 2024-12-24 11-07-37](https://github.com/user-attachments/assets/82d33fcc-bd29-4826-bcab-a99a8b4981c8)
